sql/moz-fx-data-shared-prod/adjust/adjust_derived/view.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE VIEW
+    `moz-fx-data-shared-prod.adjust_derived.adjust_derived_v1`
```

Comment:
Was the view bqetl-generated? Normally it wouldn't include the version, more like:

```suggestion
    `moz-fx-data-shared-prod.adjust_derived.adjust_derived`
```